### ISSUE TITLE
Modules pushed off screen don't return

### ISSUE THREAD

**Issue by jeffa5** [Alexays/Waybar#390]
There is an issue where modules can get 'pushed' off the end of the bar when it is filled up. 

This may be fine while there isn't enough space but the modules don't move back after there is space for them to fit as usual again. The bar has to be reloaded to reset it in my experience.

Version: `Waybar v0.6.8-9-g028b184 (Jun 12 2019, branch 'master')`

**Comment by Alexays** [comment]
May be 3c9b533997ae21ebdd09dd40c7612616fbf0beb5

**Comment by jeffa5** [comment]
The `set_hexpand` line doesn't seem to have an effect when I use it and rebuild.

**Comment by Alexays** [comment]
Can you share your config files :)

**Comment by jeffa5** [comment]
Sure, they are in this [gist](https://gist.github.com/jeffa5/932b1f0da93af4f839e5a81fdd3ce1cc).

Easiest to reproduce when using a long mode in sway I think. e.g.
```
set $mode_test some very long mode string some very long mode string some very long mode string some very long mode string some very long mode string
bindsym $mod+z mode "$mode_test"
mode "$mode_test" {
    bindsym Escape mode "default"
}
````

**Comment by Alexays** [comment]
Fixed with a6980fca7fef9b71e2cfad3855308f11e709e7d2
